player/core.h

```diff
@@ -638,6 +638,11 @@ struct mp_script_args {
     struct mpv_handle *client;
     const char *filename;
     const char *path;
+#if HAVE_PYTHON
```

Comment:
Python specific code should not be outside the python directory. Any thing in generic code should be abstracted and be the same for each scripting backed.